modules/library/coverage/src/main/java/org/geotools/coverage/Category.java

```diff
@@ -17,28 +17,27 @@
 package org.geotools.coverage;
 
 import java.awt.Color;
-import java.util.Arrays;
 import java.io.Serializable;
-
-import org.opengis.referencing.operation.MathTransform1D;
-import org.opengis.referencing.operation.TransformException;
-import org.opengis.util.InternationalString;
+import java.util.Arrays;
+import java.util.logging.Logger;
 
 import org.geotools.referencing.operation.transform.LinearTransform1D;
 import org.geotools.resources.Classes;
-import org.geotools.resources.i18n.Errors;
 import org.geotools.resources.i18n.ErrorKeys;
+import org.geotools.resources.i18n.Errors;
 import org.geotools.resources.i18n.Vocabulary;
 import org.geotools.resources.i18n.VocabularyKeys;
-import org.geotools.util.SimpleInternationalString;
 import org.geotools.util.NumberRange;
+import org.geotools.util.SimpleInternationalString;
 import org.geotools.util.Utilities;
-import org.geotools.math.XMath;
+import org.geotools.util.logging.Logging;
+import org.opengis.referencing.operation.MathTransform1D;
+import org.opengis.util.InternationalString;
 
 
 /**
  * A category delimited by a range of sample values. A categogy may be either
- * <em>qualitative</em> or <em>quantitative</em>.   For exemple, a classified
+ * <em>qualitative</em> or <em>quantitative</em>.   For example, a classified
```

Comment:
You may want to use existing pre-defined Categories as NODATA, TRUE and FALSE which have their default values and color assigned.